index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sign up page project</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<body>
+    <header>
+        <div class="infoBar">
+            <div>
+                <p>Already signed up?</p>
+                <a href="www.facebook.com"><p>Do you forget your password?</p></a>
+            </div>
+            <div class="inputbutton">
+                <input type="button" name="signin" value="Sign in"/>
+            </div>   
+        </div>
+    </header>
+    <section class="Main-header">
+        <div>
+            <h1>intuit</h1>
+            <div class="Header-logos">
+                <img src="./assets/mintLogo.png" alt="Mint" />
+                <h4>mint</h4>
+                <img src="./assets/qukBooksIcon.png" alt="qubook" />
+                <h4>quickbooks</h4>
+                <img src="./assets/Turbotax.png" alt="Turbotax" />
+                <h4>turbotax</h4>
+            </div>
+        </div>
+    </section>
+    <main>
+        <section>
+            <form>
+                <fieldset>
+                    <div class="form-header">
+                        <h2>Create an Intuit account</h2>
+                        <p>One account for everything Intuit, including Mint.</p>
+                        <a href="https://www.w3schools.com">Learn More</a>
+                    </div>
+                    <div class="form-content">
+                        <div>
+                            E-mail
+                        </div>
+                        <label>
+                            <input type="email" name="email" /> 
+                        </label>
+                        <div>
+                            Phone
+                        </div>
+                        <label>
+                            <input type="tel" name="number" /> 
+                        </label>
+                        <div>
+                            Password
+                        </div>
+                        <label>
+                            <input type="password" name="password" /> 
+                        </label> 
+                    </div>
+                    <div class="form-input-button">
+                        <input type="submit" value="Sign in" />
+                    </div>
+                    <div class="form-end">
+                        <p><i>By clicking Create account, you agree to our
+                            <a href="https://www.w3schools.com">Terms</a> and have read and acknowledge our <a href="https://www.w3schools.com">Privacy Statement</a>.</i></p>
+                    </div>
```

Comment:
This text should be centered. 
your project :arrow_down: 
![yours1](https://user-images.githubusercontent.com/32361963/83877874-eda6cb00-a743-11ea-9d5e-fa7cb758d9e6.jpg)
the clone :arrow_double_down: 
![yours2](https://user-images.githubusercontent.com/32361963/83877950-09aa6c80-a744-11ea-9cea-a7456b854e61.jpg)